eventmanager/src/components/EventsCreate.js

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+
+export function EventsCreate() {
+    const [name, setName] = useState('');
+    const [description, setDescription] = useState('');
+    const [startTime, setStartTime] = useState('');
+    const [endTime, setEndTime] = useState('');
+    const [accessCode, setAccessCode] = useState('');
+    const [group, setGroup] = useState('');
+    const [message, setMessage] = useState('');
+    const handleSubmit = async (e) => {
+        e.preventDefault();
+        try {
+            const token = localStorage.getItem('token');
+            if (!token) {
+                setMessage('No token found. Please log in.');
+                return;
+            }
+            const eventData = {
+                name,
+                description,
+                start_time: startTime,
+                end_time: endTime,
+                access_code: accessCode,
+                group,
+            };
+            const response = await axios.post('http://localhost:10001/events', eventData, {
+                headers: {
+                    Authorization: `Bearer ${token}`,
```

Comment:
Hm, if you're going to use `axios`, it could be interesting if you could look into [`interceptors`](https://axios-http.com/docs/interceptors), this way, you would automatically set t🫠he `Authorization` (and/or the `Content-Type`) header(s) on every request you make, if it is found in `localStorage`; manually setting it every time becomes painful after a while

Reply:
Thank you, it'll be done.